pysrc/scripts/sampling.py

```diff
@@ -59,8 +59,11 @@ def sample_csv(file_path: str,
             if config.header:
                 writer.writerow([file_header])
 
+            empty_str = ''
             #Changed for better readability
             for out_row in sampled_data:
+                if out_row == empty_str:
```

Comment:
Or `if out_row: writer.writerow([out_row])`

Reply:
I think I'd stay with the current version. As you said it's more verbose and easier to understand in my opinion. Just because you can do something you still have to ask whether it's good idea to improve readability.